clients/admin-ui/src/features/config-wizard/AuthenticateAwsForm.tsx

```diff
@@ -144,6 +150,13 @@ const AuthenticateAwsForm = () => {
                     // "You can find more about creating access keys and secrets on AWS docs here."
                     tooltip="The secret associated with the Access Key ID used for authentication."
                   />
+                  <CustomTextInput
+                    type="password"
+                    name="aws_session_token"
+                    label="Session Token (if applicable)"
+                    // "You can find more about creating access keys and secrets on AWS docs here."
+                    tooltip="The session token for temporary credentials."
+                  />
                   <CustomSelect
                     name="region_name"
                     label="AWS Region"
```

Comment:
This is functional, really great that you were able to get this updated and working - thank you!

one small cosmetic nit that may make the form a little easier to understand would be to mark the non-optional fields as `isRequired`, which will have them show up with an asterisk in the UI, to indicate that they're required. Then we wouldn't need the `(if applicable)` text on the new field you've added. I'd also probably tweak the tooltip text. Attaching a screenshot of what this would look like, and also providing a code suggestion to hopefully clarify (you'd also need to add `isRequired` to the `aws_acces_key_id` field, github just doesn't let me suggest that far up!):

```suggestion
                    // "You can find more about creating access keys and secrets on AWS docs here."
                    tooltip="The secret associated with the Access Key ID used for authentication."
                    isRequired
                  />
                  <CustomTextInput
                    type="password"
                    name="aws_session_token"
                    label="Session Token"
                    // "You can find more about creating access keys and secrets on AWS docs here."
                    tooltip="The session token when using temporary credentials."
                  />
                  <CustomSelect
                    isRequired
                    name="region_name"
                    label="AWS Region"
```


<img width="614" alt="image" src="https://github.com/ethyca/fides/assets/9750285/a1e68bfc-1ba4-4233-88f3-4331bb72073a">


all things considered, this form hasn't been updated in a fairly long time, and it probably needs a more general redesign that's outside of the scope of this unit of work/PR, so if you want to stick with your original implementation, that'd be fine!